src/main/java/org/zalando/nakadi/service/kpi/NakadiAccessLogPublisher.java

```diff
@@ -0,0 +1,29 @@
+package org.zalando.nakadi.service.kpi;
+
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.zalando.nakadi.service.kpi.publisher.NakadiKpiPublisher;
+
+import java.util.function.Supplier;
+
+@Service
+public class NakadiAccessLogPublisher {
+
+    private final NakadiKpiPublisher nakadiKpiPublisher;
+    private final String nakadiAccessLogEventType;
+
+    @Autowired
+    public NakadiAccessLogPublisher(
+            final NakadiKpiPublisher nakadiKpiPublisher,
+            @Value("${nakadi.kpi.event-types.nakadi.access.log:nakadi.access.log}") final String etName) {
```

Comment:
Do you plan to provide sensible default values for these properties in the `application.yml`? Are they documented somewhere? It's not evident for me what this property means by just reading its name.

Reply:
what is wrong with provided default value here?

Reply:
we do not document any other values, why this one should be exceptional?

Reply:
Forget it. I just noticed that the default is right there. It's that we sometimes provide defaults in the code and sometimes in the application yml.

Reply:
It's fine 👍 I just wonder how someone figures out that it's possible to change some property. Do people have to read the code?

Reply:
Even moving this property to application.yml won't add more understanding about the property 'value'. In my opinion it should be documented, like e.g. Kafka describes their producer, consumer and broker config params.

Reply:
I also like the idea to move this value to application.yml, it's strange to have the value right here in the code.

Reply:
moved 3686ca2